trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    super[0...10]
+  end
```

Comment:
- [ ] The name should a maximum of 10 characters. If it's longer it should trim the word. Would you kindly provide a condition that verifies the word's size? If the word is less than 10 letters, it should be returned; otherwise, it should be cut.